app/views/find-help.nunjucks

```diff
@@ -37,7 +37,7 @@
             <input type="submit" class="button" value="Search">
           </div>
         </div>
-        <p class="u-margintop5"><a href="{{ backLink.href }}" class="link-back">{{ backLink.text }}</a></p>
+        <p class="u-margintop5">&nbsp;</p>
```

Comment:
This seems a bit of hack. Any chance this could be removed? It looks like it is getting round having a `float:left` on the `.form-group` class. If I remove that and this empty `<p>` the page maintains it structure.

Reply:
Yeah, bit lazy on my part.